csharp/Platform.Data/ILinksExtensions.cs

```diff
@@ -16,13 +16,13 @@ namespace Platform.Data
     /// </summary>
     public static class ILinksExtensions
     {
-        public static TLink Create<TLink>(this ILinks<TLink, LinksConstants<TLink>> links) => links.Create(null);
+        public static TLink Create<TLink>(this ILinks<TLink, LinksConstants<TLink>> links) => links.Create(new List<TLink>());
 
         public static TLink Create<TLink>(this ILinks<TLink, LinksConstants<TLink>> links, IList<TLink> substitution)
         {
             var constants = links.Constants;
             var result = constants.Null;
-            links.Create(substitution, (_, after) =>
+            links.Create(new List<TLink>(), (_, after) =>
```

Comment:
`new List<TLink>()` is the same of performance `Array<TLink>.Empty`?

Reply:
> `new List<TLink>()` is the same of performance `Array<TLink>.Empty`?

We decided to use `List` because of our benchmarks:
![image](https://user-images.githubusercontent.com/66206278/148173207-b7db98fa-b4a6-4124-a351-9936184b4f2a.png)

Reply:
This benchmark was done to compare what is better:
1. To fill the array.
2. To fill the list.

Here it should be
```suggestion
            links.Create(substitution, (_, after) =>
```
Because you have that argument here.

Reply:
@Konard , you said that our Create substituion does not work and we shoued use an empty list instead, did not you? https://discord.com/channels/735161648448405634/782623485121265694/927138408235679767